packages/terra-core-docs/src/terra-dev-site/doc/toggle-button/example/ToggleButtonCommon.module.scss

```diff
@@ -0,0 +1,6 @@
+:local {
+  .toggle-div-style {
+    margin-top: 20px;
+    text-align: center;
```

Comment:
can we try adding padding as well so that buttons inside the container's are properly aligned

Reply:
Added fix on 'https://github.com/cerner/terra-core/pull/3827/commits/a151c1bdafa8ea0129148661770845c8e3fcbfba'